### PR TITLE
Set up base scenes for iOS

### PR DESCRIPTION
**Pull request by ShoMasegi**
## Issue
- #560 

## Overview (Required)
- Organize directories
- add basic Protocols and Extensions. (UIColor+Ext, StoryboardInstantiable ...)
- change navigation bar color and insert logo

## Screenshot
Before | After
:--: | :--:
 | 
|

### CODE REVIEW

**Comment by takahirom** [comment]
🆒😍

File: frontend/ios/DroidKaigi 2019/Protocols/StoryboardInstantiable.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    static var storyboardName: String {
+        return String(describing: self)
+    }
+
+    static func instantiateFromStoryboard() -> Self {
+        let storyboard = UIStoryboard(name: storyboardName, bundle: nil)
+        let controller = storyboard.instantiateInitialViewController() as? Self
+        if controller == nil {
+            assert(false, "生成したいViewControllerと同じ名前のStorybaordが見つからないか、Initial ViewControllerに設定されていない可能性があります。")
```
Comment (takahirom) [inline frontend/ios/DroidKaigi 2019/Protocols/StoryboardInstantiable.swift]: Can you translate?🙏

Comment (ShoMasegi) [reply frontend/ios/DroidKaigi 2019/Protocols/StoryboardInstantiable.swift]: Sorry! I Fixed!🙇‍♂️

**Comment by takahirom** [comment]
Can you resolve conflict?👀

**Comment by ShoMasegi** [comment]
@takahirom Sorry, I cannot resolve this rename conflict😥On my side, There is no conflict....

**Comment by KeithYokoma** [comment]
@ShoMasegi I could happen when you use newer `git` version than the one used in GitHub. If it succeed to merge in your local and can push it to your branch, then GitHub will mark it resolved.

**Comment by ShoMasegi** [comment]
@KeithYokoma Thanks!! It works well!!😇

**Comment by takahirom** [comment]
Thank you!

**Pull request merged**